loaders/dashboards/pkg/util/grafana_util_test.go

```diff
@@ -32,7 +33,7 @@ func createFakeServer(t *testing.T) {
 	)
 	err := http.ListenAndServe(":3002", server3002)
 	if err != nil {
-		t.Fatal("fail to create internal server at 3002")
```

Comment:
Same as my other comment: `*testing.T.Fatal` is very different from `log.Fatal`. I don't think this should be changed.